app/src/pages/inside/settingsPage/settingsTabs/autoAnalysisTab/analysisForm/formField/formField.jsx

```diff
@@ -0,0 +1,90 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { FieldProvider } from 'components/fields/fieldProvider';
+import { FieldErrorHint } from 'components/fields/fieldErrorHint';
+import styles from './formField.scss';
+
+const cx = classNames.bind(styles);
+
+export class FormField extends PureComponent {
```

Comment:
I suggest moving this component to common components since there are other forms with the same structure.
@ArtsiomYeliseyenka what do you think?

Reply:
Moved